assemblies/guardrails-orchestrator-service.adoc

```diff
@@ -0,0 +1,27 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+[id="guardrails_orchestrator_service_{context}"]
+= Guardrails orchestrator service
+
```

Comment:
capital O? - for consistency with text in paragraph
Guardrails Orchestrator service